src/interface/AbstractPortal.sol

```diff
@@ -59,15 +68,21 @@ abstract contract AbstractPortal is Initializable, IERC165Upgradeable {
     // Run all modules for all payloads
     moduleRegistry.bulkRunModules(modules, attestationsPayloads, validationPayloads);
     // Register attestations using the attestation registry
-    attestationRegistry.bulkAttest(attestationsPayloads);
+    attestationRegistry.bulkAttest(attestationsPayloads, _getAttester());
   }
 
   /**
    * @notice Revokes the attestation for the given identifier and can replace it by a new one
    * @param attestationId the attestation ID to revoke
-   * @param replacedBy the replacing attestation ID (leave to just revoke)
+   * @param replacedBy the replacing attestation ID (leave empty to just revoke)
+   * @dev By default, revocation is only possible for the attestation issuer (attester) or the portal owner
+   * We strongly encourage implementing such a rule in your Portal if you intend on overriding this method
    */
   function revoke(bytes32 attestationId, bytes32 replacedBy) external virtual {
```

Comment:
I think for most use cases, the only a person who should be able to revoke an attestation is the portal owner, i.e.:

`if (msg.sender != portalRegistry.getPortalByAddress(address(this)).ownerAddress)) revert OnlyAttesterOrOwner();`

For other use cases, this can be changed of course, but I think the code above is a fairly good opinionated default

Reply:
OK! And this costs less gas! 👍